channels/client/addin.h

```diff
@@ -18,3 +18,18 @@
  */
 
 
+/* we need rdpContext */
+//#include <freerdp/freerdp.h>
+
+void
+channel_client_create_handler(rdpContext *ctx, HANDLE thread, LPVOID userdata,
+	UINT (*msg_handler)(LPVOID userdata,  wStream* data), wMessageQueue* queue);
+
+UINT
+channel_client_post_message(rdpContext *ctx, LPVOID pData, UINT32 dataLength,
+	UINT32 totalLength, UINT32 dataFlags, wStream* channel_data_in,
+	wMessageQueue* queue, UINT (*msg_handler)(LPVOID userdata,  wStream* data),
```

Comment:
Why do you add the function pointer for each post? Should be sufficient to add it during `channel_client_create_handler` or do I miss something?

Reply:
it was just draft, to confirm what we understand each other, look at latest changes.